beluga_amcl/include/beluga_amcl/common.hpp

```diff
@@ -0,0 +1,466 @@
+// Copyright 2024 Ekumen, Inc.
```

Comment:
@serraramiro1 nit: perhaps name this file so it is clear it is ROS 2 specific?

Reply:
Done